slicedimage/io.py

```diff
@@ -188,8 +188,9 @@ def parse(self, json_doc, baseurl):
                         sha256=tile_doc.get(TileKeys.SHA256, None),
                         extras=tile_doc.get(TileKeys.EXTRAS, None),
                     )
+                    seekable = True if tile_format == ImageFormat.NUMPY else False
```

Comment:
Please add a comment here explaining what this is and why we need to set this flag.

Reply:
I moved this into the ImageFormat enum, and added a comment on the accessor.